GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist

```diff
@@ -2,6 +2,46 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
It looks like this is the only plist file in the glyphs sub-directory.  Intentional? Or is this the incorrect path?

Reply:
I haven’t created this structure, I’m only following it. So far there has only been one file. It is the same in the other writing system directories.

The CustomFilter file has each glyphset separate anyway:
<img width="266" alt="Screenshot 2023-11-07 at 11 24 00" src="https://github.com/googlefonts/glyphsets/assets/1923747/e687ba60-ec64-4850-9b86-e55937b5a374">

Reply:
Copy that.  Ty!